src/main/java/edu/kit/datamanager/repo/util/validators/impl/HandleNetValidator.java

```diff
@@ -0,0 +1,156 @@
+ */
+public class HandleNetValidator implements IValidator {
+
+    Logger log = LoggerFactory.getLogger(HandleNetValidator.class);
+
+    /**
+     * This method returns the type of the validator implementation.
+     *
+     * @return element of the enum defined in the Datacite schema.
+     */
+    @Override
+    public RelatedIdentifierType supportedType() {
+        return RelatedIdentifierType.HANDLE;
+    }
+
+    /**
+     * This method must be implemented by any implementation.
+     * It validates an input and either returns true or throws an exception.
+     *
+     * @param input to validate
+     * @param type  of the input
+     * @return true if input is valid for the special type of implementation.
+     */
+    @Override
+    public boolean isValid(String input, RelatedIdentifierType type){
+        if (type != supportedType()) {
+            LOG.warn("Illegal type of validator");
+            throw new UnsupportedMediaTypeException("Illegal type of Validator.");
+        }
+
+        String regex = "^(hdl://|http://|https://|doi:)(.+)";
+        Pattern pattern = Pattern.compile(regex, Pattern.MULTILINE);
+        Matcher matcher = pattern.matcher(input);
+        if (matcher.find()) {
+            if (matcher.group(1).equals("https://") || matcher.group(1).equals("http://"))
+                return isValidHTTPURL(matcher.group(0));
+            else return isValidHandle(matcher.group(2));
+        } else return isValidHandle(input);
+    }
+
+    /**
+     * This method checks if the given prefix and suffix is downloadable.
+     * If there is some invalid input, it will validate the prefix with help of the RESTful API by handle.net.
+     *
+     * @param serverAddress from the server which should be uses for the validation.
+     * @param prefix        the handle prefix
+     * @param suffix        the handle suffix
+     * @return true if the record is downloadable
+     */
+    private boolean isDownloadable(String serverAddress, String prefix, String suffix) {
+        IValidator urlValidator = new URLValidator();
+        AtomicBoolean fullValid = new AtomicBoolean(false);
+        log.debug("Server address: {}", serverAddress);
+        log.debug("Prefix: {}", prefix);
+        log.debug("Suffix: {}", suffix);
+        ignoringExc(() -> {
+            try{
+                fullValid.set(urlValidator.isValid(serverAddress + "/" + prefix + "/" + suffix));
+            } catch (Exception ignored){
+            }
+        });
+        if (fullValid.get()) {
+            LOG.info("The handle is valid!");
+            return true;
+        }
+        log.warn("Either the suffix or the prefix might be invalid. Proving if prefix is valid...");
+        if (urlValidator.isValid("https://hdl.handle.net/0.NA/" + prefix)) {
+            log.info("The prefix {} is valid!", prefix);
+            throw new MessageValidationException("Prefix valid, but suffix not");
+        }
+        log.error("The entered prefix is invalid!");
```

Comment:
You have to provide additional infos here. If log level is set to ERROR not further information is provided which is very frustrating. In case of an error you should provide all information so that the administrator can understand the problem.
(If there is any (s)he could fix)